### PR TITLE
Orangelight

### PR DESCRIPTION
**Pull request by kayiwa**
Hey @tampakis thanks for looking into this. We need an easy way to stand up an orangelight node. This borrows HEAVILY from @tpendragon LAE (now dead) branch.

### CODE REVIEW

**Comment by eliotjordan** [comment]
In general, this looks good. I'm having some difficulty getting this to run on my local Vagrant environment. Not sure if this is an issue on my machine, or if others have this problem as well.

**Comment by kayiwa** [comment]
hey @eliotjordan what version of ansible do you have? 
```
ansible --version
```

**Comment by eliotjordan** [comment]
ansible 2.4.0.0

**Comment by eliotjordan** [comment]
Should we include the necessary ENV variables in this PR? OL_FEEDBACK_TO, OL_FEEDBACK_CC, SOLR_URL, etc...

**Comment by eliotjordan** [comment]
Ping @tampakis ^

File: roles/pulibrary.capistrano/files/libimages1.smb.credentials [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$ANSIBLE_VAULT;1.1;AES256
```
Comment (tampakis) [inline roles/pulibrary.capistrano/files/libimages1.smb.credentials]: Orangelight doesn't need files like this. Is it ok to remove or are other apps going to use this role?

Comment (escowles) [reply roles/pulibrary.capistrano/files/libimages1.smb.credentials]: I think it's fine to remove them — Figgy has its own copies: https://github.com/pulibrary/princeton_ansible/tree/master/roles/pulibrary.figgy/files

**Comment by kayiwa** [comment]
as an aside you `may` want to create a Vagrantfile for this since with the most recent merge the Vagrantfile is now ignored and a symbolic link is expected. Look at `Vagrant/apacheVagrantfile` for examples/guidance... although in orangelight's case I saw little reason for the need of a Vagrantfile use.

**Comment by tampakis** [comment]
Should be good to go. Merge conflicts with vault is a real pain!

**Comment by kayiwa** [comment]
oh man. Did you try `ansible_vault-merge.sh` ?

**Pull request merged**